misk/web/tabs/web-actions/src/ducks/webActions.ts

```diff
@@ -159,7 +159,7 @@ function* handleMetadata() {
             )
         )
         const emptyAllowedArrayValue =
-          authFunctionAnnotations.length > 1 &&
+          authFunctionAnnotations.length > 0 &&
           authFunctionAnnotations[0].includes("Unauthenticated")
```

Comment:
above we only filter `authFunctionAnnotations` to include "Access" or "authz", would this ever be true then?

Reply:
I'm not sure if we have a strict one `Access` or `authz` annotation per web action, if it is enforced as strictly one then we can restructure that processing so it assumes there will only ever be one match.

Reply:
let's sync up offline about this